vsg/__main__.py

```diff
@@ -33,6 +33,7 @@ def parse_command_line_arguments():
     parser.add_argument('-b', '--backup', default=False, action='store_true', help='Creates a copy of input file for comparison with fixed version.')
     parser.add_argument('-oc', '--output_configuration', default=None, action='store', help='Write configuration to file name.')
     parser.add_argument('-rc', '--rule_configuration', default=None, action='store', help='Display configuration of a rule')
+    parser.add_argument('--style', action='store', choices=get_installed_styles(), help='Uses built in style')
```

Comment:
Here https://github.com/jeremiah-c-leary/vhdl-style-guide/issues/350 `built in configuration` name is used. In the code and documentation `style`. I propose to use one term, `Built-in Styles`. Of course the argument stays as `--style`, `--built-in-style` would be too long.

Reply:
I eventually went with `--style` because `--built-in-configuration` was too long.

Maybe instead of `Uses built in style` it should be `Use predefined style` or `Use available style`?

Reply:
I like predefined.